docs/blocks/feature-flags-and-experimental-interfaces.md

```diff
@@ -45,13 +45,14 @@ We also have individual features or code blocks behind a feature flag, this is a
 ## Processes and commands that use a flag
 
 -   `npm run build:deploy` uses the feature plugin flag ([env flag](https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/c0de18ec0a798c072420c67a689e4cc4d3ac77c9/package.json#L28)).
--   travis uses the experimental flag when running tests ([env flags](https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/d6a506c19056d42d40743ae0126153fb81004672/.travis.yml#L68-L143)).
+-   GitHub actions uses the experimental flag when running automated tests ([env flags](https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/trunk/.github/workflows/php-js-e2e-tests.yml)).
```

Comment:
Would it help if we added a code line to the link? Not sure if it's the right line tho` `https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/trunk/.github/workflows/php-js-e2e-tests.yml#L263`

Reply:
That flag is used in several places in that file, so I didn't want to just link to one instance, for fear of implying that it was an isolated use. We could link to each use actually, and very explicitly document each individual use. What do you think?

Reply:
I think that would be hard to maintain. It's ok to leave it as it is.